mmv1/products/identityplatform/api.yaml

```diff
@@ -505,3 +505,94 @@ objects:
           Whether authentication is disabled for the tenant. If true, the users under
           the disabled tenant are not allowed to sign-in. Admins of the disabled tenant
           are not able to manage its users.
+  - !ruby/object:Api::Resource
+    name: ProjectDefaultConfig
+    base_url: 'projects/{{project}}/config'
+    self_link: 'projects/{{project}}/config'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      There is no persistent data associated with this resource.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The name of the Config resource. Example: "projects/my-awesome-project/config"
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'signIn'
+        description: |
+         Configuration related to local sign in methods.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: email 
+            description: |
+              Configuration options related to authenticating a user by their email address.
+            properties:
+            - !ruby/object:Api::Type::Boolean
+              name: enabled
+              description: |
+                Whether email auth is enabled for the project or not.
+            - !ruby/object:Api::Type::Boolean
+              name: 'passwordRequired'
+              description: |
+                Whether a password is required for email auth or not. If true, both an email and 
+                password must be provided to sign in. If false, a user may sign in via either 
+                email/password or email link.
+          - !ruby/object:Api::Type::NestedObject
+            name: phoneNumber 
+            description: |
+              Configuration options related to authenticated a user by their phone number.
+            properties:
+            - !ruby/object:Api::Type::Boolean
+              name: enabled
+              description: |
+                Whether phone number auth is enabled for the project or not.
+            - !ruby/object:Api::Type::KeyValuePairs
+              name: 'testPhoneNumbers'
+              description: |
+                A map of <test phone number, fake code> that can be used for phone auth testing.
+          - !ruby/object:Api::Type::NestedObject
+            name: anonymous 
+            description: |
+              Configuration options related to authenticating an anonymous user.
+            properties:
+            - !ruby/object:Api::Type::Boolean
+              name: enabled
```

Comment:
This should be required since it's the only attribute in the nested block.

Reply:
Done